server/src/internalClusterTest/java/org/elasticsearch/cluster/SpecificMasterNodesIT.java

```diff
@@ -103,10 +103,6 @@ public void testElectOnlyBetweenMasterNodes() throws Exception {
             internalCluster().nonMasterClient().admin().cluster().prepareState().get().getState().nodes().getMasterNode().getName(),
             equalTo(masterNodeName)
         );
-        assertThat(
-            internalCluster().nonMasterClient().admin().cluster().prepareState().get().getState().nodes().getMasterNode().getName(),
-            equalTo(masterNodeName)
-        );
```

Comment:
Seems like a duplicate of the previous check.

I was looking in here because I broke the test momentarily.